packages/account/src/Sections/Security/DeactivateAccount/account-has-balance.jsx

```diff
@@ -0,0 +1,165 @@
+import React from 'react';
+import { Button, Icon, Money } from '@deriv/components';
+import { formatMoney } from '@deriv/shared';
+import { localize } from '@deriv/translations';
+
+const getMT5AccountType = (group) => (group ? group.replace('\\', '_').replace(/_(\d+|master|EUR|GBP)/, '') : '');
+
+const getMT5AccountDisplay = (group) => {
+    if (!group) return {};
+
+    const value = getMT5AccountType(group);
+    let display_text = localize('MT5');
+    if (/svg$/.test(value)) {
+        display_text = localize('Synthetic');
+    } else if (/vanuatu/.test(value) || /svg_(standard|financial)/.test(value)) {
+        display_text = localize('Financial');
+    } else if (/labuan/.test(value)) {
+        display_text = localize('Financial STP');
+    }
+
+    return display_text;
+};
+
```

Comment:
Is it possible to get this from `/deriv-app/packages/core/src/Stores/Helpers/client.js`?

Reply:
do we have access to this file from here?